mantle/cmd/kola/testiso.go

```diff
@@ -388,11 +389,8 @@ func runTestIso(cmd *cobra.Command, args []string) error {
 		ranTest = true
 		instPxe := baseInst // Pretend this is Rust and I wrote .copy()
 
-		if err := testPXE(ctx, instPxe, filepath.Join(outputDir, scenarioPXEInstall), false); err != nil {
-			return errors.Wrapf(err, "scenario %s", scenarioPXEInstall)
-
-		}
-		printSuccess(scenarioPXEInstall)
+		err := testPXE(ctx, instPxe, filepath.Join(outputDir, scenarioPXEInstall), false)
+		failed = printResult(scenarioPXEInstall, err) || failed
```

Comment:
I think it'd be cleaner to maintain an error list, then fail if that list has anything non-`nil`.

Reply:
Would we do anything with the error objects? We already print each one in `printResult`?